examples/2-tier-iis-sql-vm/modules/network/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable resource_group_name {}
+variable environment_name {}
+variable location {}
+variable address_space {}
+
+variable dns_servers {
+  type = "list"
+}
+
+variable wafsubnet_name {}
+variable wafsubnet_prefix {}
+variable rpsubnet_name {}
+variable rpsubnet_prefix {}
+variable issubnet_name {}
+variable issubnet_prefix {}
+variable dbsubnet_name {}
+variable dbsubnet_prefix {}
+variable dcsubnet_name {}
+variable dcsubnet_prefix {}
```

Comment:
this syntax (defining a variable without the name being in quotes) will break in Terraform 0.12 - it'd be worth updating these to be quoted to ensure this remains compatible in the future